tests/app/service/test_rest.py

```diff
@@ -2051,3 +2050,34 @@ def test_search_for_notification_by_to_field_filters_by_statuses(client, notify_
     assert len(notifications) == 2
     assert str(notification1.id) in notification_ids
     assert str(notification2.id) in notification_ids
+
+
+def test_search_for_notification_by_to_field_returns_content(
+    client,
+    notify_db,
+    notify_db_session,
+    sample_template_with_placeholders
+):
+    notification = create_sample_notification(
+        notify_db,
+        notify_db_session,
+        to_field='+447700900855',
+        normalised_to='447700900855',
+        template=sample_template_with_placeholders,
+        personalisation={"name": "Foo"}
+    )
+
+    response = client.get(
+        '/service/{}/notifications?to={}'.format(
+            sample_template_with_placeholders.service_id, '+447700900855'
+        ),
+        headers=[create_authorization_header()]
+    )
+    notifications = json.loads(response.get_data(as_text=True))['notifications']
+
+    assert response.status_code == 200
+    assert len(notifications) == 1
+
+    assert notifications[0]['id'] == str(notification.id)
+    assert notifications[0]['to'] == '+447700900855'
+    assert notifications[0]['body'] == 'Hello Foo\nYour thing is due soon'
```

Comment:
We should keep using the schema in the same way for consistency. 👍 to not change.